frontend/src/features/properties/map/acquisition/AcquisitionContainer.tsx

```diff
@@ -107,12 +129,14 @@ export const AcquisitionContainer: React.FunctionComponent<IAcquisitionContainer
     >
       <FileLayout
         leftComponent={
-          <AcquisitionMenu
-            items={menuItems}
-            selectedIndex={containerState.selectedMenuIndex}
-            onChange={onMenuChange}
-            setContainerState={setContainerState}
-          />
+          <>
+            <AcquisitionMenu
+              items={menuItems}
+              selectedIndex={containerState.selectedMenuIndex}
+              onChange={onMenuChange}
+              setContainerState={setContainerState}
+            />
+          </>
```

Comment:
out of curiosity - why do we need the extra `<></>` wrapper if there's only a single child component?